points_topic: "/tree_points"
cylinder_topic: "/tree_trunks"
robot_vis_topic: "/robot_cov"
landmark_vis_topic: "/landmarks"